Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -112,6 +123,9 @@ public InputOverlay(Context context, AttributeSet attrs)
     if (!mPreferences.getBoolean("OverlayInitV3", false))
       defaultOverlay();
 
+    overlayOpacity = mPreferences.getInt("OverlayOpacity", OVERLAY_DEFAULT_OPACITY);
+    autoDim = mPreferences.getBoolean("autoDimOverlay", true);
```

Comment:
Isn't it unnecessary to do this here since it also gets done in `refreshControls`?